tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php

```diff
@@ -74,6 +76,15 @@ public function testRegisterExternalBucketInBYODBEnvironment(): void
                 SnowflakeQuote::quoteSingleIdentifier(self::TEST_TABLE),
             ),
         );
+        $db->executeQuery('USE WAREHOUSE DEV;');
+
+        $db->executeQuery(
```

Comment:
chci assertnout, ze to pak realne udela nejaka data. usecase, kdy se testuje WS jako EB a jeho load je testovan v SnowflakeRegisterBucketTest::testRegisterWSAsExternalBucket